ProcessLib/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -176,40 +162,11 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
             "reference_temperature", std::numeric_limits<double>::quiet_NaN());
     DBUG("Use 'reference_temperature' as reference temperature.");
 
-    //Specific gas constant
-    double const specific_gas_constant =
-        //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__specific_gas_constant}
-        config.getConfigParameter<double>(
-            "specific_gas_constant", std::numeric_limits<double>::quiet_NaN());
-    DBUG("Use 'specific_gas_constant' as specific gas constant.");
-
-    // Fluid compressibility
-    double const fluid_compressibility =
-        //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__fluid_compressibility}
-        config.getConfigParameter<double>(
-            "fluid_compressibility", std::numeric_limits<double>::quiet_NaN());
-    DBUG("Use 'fluid_compressibility' as fluid compressibility parameter.");
-
-    auto const fluid_type =
-        FluidType::strToFluidType(
-            //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS__fluid_type}
-            config.getConfigParameter<std::string>("fluid_type"));
-    DBUG("Use 'fluid_type' as fluid type parameter.");
-
-    if (!FluidType::checkRequiredParams(fluid_type, fluid_compressibility,
-                                        reference_temperature,
-                                        specific_gas_constant))
-    {
-        OGS_FATAL(FluidType::getErrorMsg(fluid_type));
-    }
-
     HydroMechanicsProcessData<DisplacementDim> process_data{
         materialIDs(mesh),     std::move(media_map),
         std::move(solid_constitutive_relations),
-        initial_stress,        intrinsic_permeability,
-        fluid_density,         specific_body_force,
-        fluid_compressibility, reference_temperature,
-        specific_gas_constant, fluid_type};
+        initial_stress,        specific_body_force,
+        reference_temperature};
```

Comment:
The reference temperature is also available in MPL... https://github.com/ufz/ogs/blob/master/MaterialLib/MPL/PropertyType.h#L64

Reply:
also add this a medium property?

Reply:
Yes.